examples/Table-nested-items.rb

```diff
@@ -100,6 +105,15 @@ def update_selected(id)
       id ||= "<nil>"
       UI.ChangeWidget(Id(:selected), :Value, id.to_s)
     end
+
+    def open_items
+      UI.QueryWidget(Id(:table), :OpenItems).keys
+    end
+
+    def update_open_items(ids)
+      ids ||= "<nil>"
```

Comment:
aren't we reinventing a wheel, in particular `ids.inspect` ?

Reply:
Right; we can also simply use plain Ruby `inspect()` here.